MyHeartCounts.xcodeproj/project.pbxproj

```diff
@@ -1510,7 +1510,15 @@
 			repositoryURL = "https://github.com/StanfordSpezi/SpeziFoundation.git";
 			requirement = {
 				kind = upToNextMajorVersion;
-				minimumVersion = 2.4.1;
+				minimumVersion = 2.4.4;
+			};
+		};
+		802FA99B2EACF24A00624C31 /* XCRemoteSwiftPackageReference "SpeziFirebase" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/StanfordSpezi/SpeziFirebase.git";
+			requirement = {
+				kind = revision;
+				revision = 3a00b391f39104444465da3b9105a4c75dc1c748;
```

Comment:
TOOD: switch to tagged release once https://github.com/StanfordSpezi/SpeziFirebase/pull/59 is merged